README.rst

```diff
@@ -106,11 +106,12 @@ view fails for some reason.
 The bin/upgrade script
 ======================
 
-Refer to the `console script installation`_ section how to install ``bin/upgrade``.
```

Comment:
there is a hyperlink target `console script installation` before the title you are now using. if you no longer use the target please remove it, too.

Reply:
Ah - I missed that. I was trying to fix the hyperlink that is broken on Github (not Pypi), maybe quotes on the target will do it? Otherwise I will not interfere with that.

Reply:
Ok - I missed that target - have restored the link